src/transformers/keras_callbacks.py

```diff
@@ -126,6 +137,11 @@ def __init__(
         if parse(tf.__version__) < parse("2.7"):
             logging.warning("TF versions less than 2.7 may encounter issues with KerasMetricCallback!")
 
+        self.use_xla_generation = use_xla_generation
+        self.generate_kwargs = generate_kwargs or {}
```

Comment:
We don't use `or` in Transformers like this. Please be explicit with a one-line test.